test/ggsn_SUITE.erl

```diff
@@ -154,6 +154,8 @@
 		     {ip_pools, ['pool-A']}]}
 		  ]},
 
+		 {teid, {3, 6}},
+
 		 {charging,
```

Comment:
does the test work without that? the default for teid should have worked (but I didn't test that)

Reply:
Without this setting `{3, 6}` the tests is failed

Reply:
it should work without, the default is {0,0}.
Can you find out why it failes?

Reply:
Looks like the value `{0, 0}` is not set by default.

Reply:
The default value cannot be set by reason that in `ergw_app:start/2` we call `Pid <- ergw_sup:start_link(),`(where we start `ergw_tei_mngr` - when we call  `ergw_tei_mngr` and try get in the function `init` the configurations, the config is not exist and default value is not set) and then we call `ergw_config:load_config(setup:get_all_env(ergw))` where we are set the default value in configuration. Plus we need also update `ergw:load_config/1` for set default value for new options. Not sure, but looks like the `init` function is not a best plays for getting config of `Prefix, PrefixLen` in current case.

Reply:
Fixed.